broker-util/oo-admin-repair

```diff
@@ -930,15 +923,29 @@ end
 def repair_orphaned_envs(orphaned_envs)
   return if orphaned_envs.blank?
   failed_count = 0
-  orphaned_envs.each do |orphan|
-    domain = Domain.find(orphan[:domain_id])
+  puts "Removing orphaned environment variables from domain(s)..." if $verbose
+  domain_component = orphaned_envs.group_by{ |d| d[:domain_id] }.each{ |_, v| v.map!{ |d| d[:component_id] }}
+  domain_component.each do |domain_key, component_array|
     begin
-      domain.remove_env_variables(orphan[:component_id])
-      puts "Removed envs with component id #{orphan[:component_id]} from domain \"#{domain.namespace}\""
-    rescue => e
-      print_message "Unable to remove envs with component id #{orphan[:component_id]} from domain \"#{domain.namespace}\": #{e.message}"
+      variables = []
```

Comment:
This is not a very descriptive name for this variable. Perhaps something like `env_vars = []`.

Calling any variable `variable` is generally a bad idea.